apps/src/tutorialExplorer/tutorialDetail.jsx

```diff
@@ -200,7 +250,11 @@ export default class TutorialDetail extends React.Component {
                       rel="noopener noreferrer"
                       onClick={this.startTutorialClicked}
                     >
-                      <button type="button" style={{marginTop: 20}}>
+                      <button
+                        type="button"
+                        style={{marginTop: 20}}
+                        tabIndex={-1}
```

Comment:
The button wrapped in a link is kinda weird, but I left it for now. Regardless, I removed the inner button from the tab order so we don't tab to it twice.

Reply:
> Is there a reason why it's wrapped in this manner? And, if not, would we have a preference between using just the button or just the link? I'd lean towards the button with an `onClick` handler, but a stylized link would also be fine.

I don't think it needs to stay this way, and was just done so that we'd have something that acted like a link but looked like a button.  I think it's best to leave the `<a>` intact so that screen-readers and browsers themselves get a proper preview of the target URL, but to just have a `<div>` styled to look like a button.  I prepped such a fix in the same [diff](https://github.com/code-dot-org/code-dot-org/pull/51365/commits/e2bcd2f4aa6244629d6eb320c290054686f19888?w=1) mentioned elsewhere.